contrib/intel/jenkins/Jenkinsfile

```diff
@@ -245,6 +245,50 @@ pipeline {
                         }
                     }
                 }
+                stage('psm3') {
+                    agent {node {label 'mlx5'}}
+                    options { skipDefaultCheckout() }
+                    steps {
+                        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                        {
+                          sh """
+                            env
+                            (
+                                export PSM3_IDENTIFY=1
+                                export FI_LOG_LEVEL=info
+                                cd /opt${env.WORKSPACE}/contrib/intel/jenkins/
+                                python3.7 runtests.py --prov=psm3 --test=fabtests
+                                python3.7 runtests.py --prov=psm3 --test=fabtests --ofi_build_mode=dbg
+                                python3.7 runtests.py --prov=psm3 --test=fabtests --ofi_build_mode=dl
+                                echo "psm3 completed."
+                            )
+                          """
+                        }
+                    }
+                }
+                stage('MPI-psm3') {
+                    agent {node {label 'mlx5'}}
+                    options { skipDefaultCheckout() }
+                    steps {
+                        withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                        {
+                          sh """
+                            env
+                            (
+                                export PSM3_IDENTIFY=1
+                                cd /opt${env.WORKSPACE}/contrib/intel/jenkins/
+                                python3.7 runtests.py --prov=psm3 --test=IMB --imb_grp=1
+                                echo "IMB psm3 Group 1 completed."
+                                python3.7 runtests.py --prov=psm3 --test=IMB --imb_grp=2
+                                echo "IMB psm3 Group 2 completed."
+                                python3.7 runtests.py --prov=psm3 --test=IMB --imb_grp=3
+                                echo "IMB psm3 Group 3 completed."
+                                echo "MPI-psm3 completed."
```

Comment:
@shefty Do we want to test all MPI tests with psm3? This includes the MPI1, RMA, IO, EXT, NBC, P2P, and MT test suites

Reply:
psm3 is developed out of tree with massive code dumps every 3 months.  I see minimal to no value in per PR CI testing psm3 at all when the code is static.  Other than that I really don't care what we do if the CI completes successfully and quickly.

Reply:
Is it possible to maybe detect if a provider or core gets updated and only enabled the needed tests?
For instance, if only prov/verbs/* gets updated only run verbs; if core gets update run all, etc.

Do we run a longer nightly/weekly test?

Reply:
We are moving to the model that CI is skipped for irrelevant changes.  Jenkins just doesn't make detecting this easy.  We are removing nightly/weekly tests because they occur too long after the fact and problems detected there require someone unfamiliar with the code to isolate the problem and report it back to the original submitter.  But by then fixes are no longer urgent, since the PR has already been merged.